CMakeLists.txt

```diff
@@ -491,41 +491,47 @@ endif(UNIX)
 
     if(UNIX)
 
-        if(ANTILIB_PATH)
-            ADD_DEFINITIONS(-DANTILIB_PATH="${ANTILIB_PATH}")
+        if(ANTIMICROLIB_PATH)
+            ADD_DEFINITIONS(-DANTIMICROLIB_PATH="${ANTIMICROLIB_PATH}")
 
             # Generally package manager takes care of this while installing the new library, but not always
             # especially with cmake
-            ADD_DEFINITIONS(-DLD_LIBRARY_PATH="${ANTILIB_PATH}:${LD_LIBRARY_PATH}")
+            ADD_DEFINITIONS(-DLD_LIBRARY_PATH="${ANTIMICROLIB_PATH}:${LD_LIBRARY_PATH}")
 
             # those instructions have to be before add_executable() or add_library() is called, otherwise it has no effect
-            set(CMAKE_INSTALL_RPATH "${ANTILIB_PATH}")
+            set(CMAKE_INSTALL_RPATH "${ANTIMICROLIB_PATH}")
             set(CMAKE_INSTALL_RPATH_USE_LINK_PATH TRUE)
         endif()
 
-            add_library( antilib
+            add_library( antimicrolib
                 SHARED
                 ${antimicrox_HEADERS_MOC}
                 ${antimicrox_SOURCES}
                 ${antimicrox_FORMS_HEADERS}
                 ${antimicrox_RESOURCES_RCC}
             )
 
-            set_target_properties(antilib PROPERTIES
-                SOVERSION ${ANTILIB_VERSION}
+            set_target_properties(antimicrolib PROPERTIES
+                NO_SONAME 1
             )
 
+            set_target_properties(antimicrolib PROPERTIES
+                OUTPUT_NAME "antimicro"
+            )
```

Comment:
"lib" prefix will be added automatically, renaming it so the name will be "libantimicro"

Reply:
and on other platforms the corresponding naming conventions will apply